server/models.py

```diff
@@ -13,33 +23,44 @@ class Author(db.Model):
     isAdmin = db.Column(db.Boolean())
     isVerified = db.Column(db.Boolean())
 
-
-    def __init__(self, githubId, profileImageId, displayName, isAdmin = False, isVerified = True):
+    def __init__(self, githubId, profileImageId, displayName, isAdmin=False, isVerified=True):
         self.githubId = githubId
         self.profileImageId = profileImageId
         self.displayName = displayName
         self.isAdmin = isAdmin
-        self.isVerified = isVerified#should set this from a JSON or db later
-
+        self.isVerified = isVerified  # should set this from a JSON or db later
 
     def __repr__(self):
         return f"<id {self.id}>"
 
+    def json(self) -> Dict[str, str]:
+        # get username from github id
+        resp = urlopen(f'https://api.github.com/user/{self.githubId}')

```

Comment:
If the github ID is invalid this raises a 404 ERROR exception. For our side we should expect that unless were creating a debug or testing author this wont be an issue but I wonder about when we have to hook our API to other groups. If we fix this I suggest only calling `author.json()` in a try except block.

Reply:
Yeah, when we hook it up to other groups then we'll have to look at making this more resilient but for now, I think it's ok.